_posts/2023-11-03-db-benchmark-update.md

```diff
@@ -0,0 +1,50 @@
+	
+In order to be fair to all solutions, we decided to change the instance type to a metal instance with instance storage. Metal instance types negate any noisy neighbor problems because the hardware is physical and not shared with any other AWS users/instances. Network storage problems are also fixed because solutions can read and write data to the local instance storage, which is physically mounted on the hardware.
+
+Another benefit of the the c6id.metal box is that it stresses parallelism performance. There are 128 cores on the c6id.metal. Performance differences between solutions that can effectively use every core and solutions that cannot are clearly visible.
+
+See the [updated settings](#updated-settings) section on how settings were change for each solution when run on the new machine.
+
+## Updating the Benchmark
+
+Moving forward we will update the benchmark when PRs with new performance numbers are provided. The PR should include a description of the changes to a solution script or a version update and new entries in the `time.csv` and `logs.csv` files. These entries will be verified using a different c6id.metal instance, and if there is no variance, the PR will be merged and the results will be updated!
+
+### Updated Settings
+
+1. ClickHouse
+	- Storage: Any data this gets spilled to disk also needs to be on the NVMe drive. This has been changed, see the `format_and_mount.sh` script, and the `clickhouse/clickhouse-mount-config.xml` files on how this is done.
+2. Julia (juliadf & juliads)
+	- Threads: The threads were hardcoded for juliadf/juliads to 20/40 threads. Now the max number of threads are used. No option was given to spill to disk, so this was not changed/researched.
+3. DuckDB
+	- Storage: The DuckDB database file was specified to run on the NVMe mount.
+4. Spark
+	- Storage: There is an option to spill to disk. I was unsure of how to modify the storage location so that it was on the NVMe drive. Open to a PR with storage location changes and improved results!
+
+Many solutions do not spill to disk, so they did not require any modification to use the instance storage. Other solutions use `parallel::ncores()` or default to a maximum number of cores for parallelism. Solution scripts were run in their current form on [github.com/duckdblabs/db-benchmark](https://github.com/duckdblabs/db-benchmark). Please read the [Updating the Benchmark](https://github.com/duckdblabs/db-benchmark#updating-the-benchmark) section on how to re-run your solution.
```

Comment:
@Tmonster the link to the "Updating the Benchmark" section is broken

Reply:
this should work once I merge this PR https://github.com/Tmonster/db-benchmark/pull/6 to duckdblabs/db-benchmark.